module/server.cpp

```diff
@@ -9,9 +9,17 @@
 //#include "modules/gdnative/gdnative.h"
 #include "gdnative.h"
 
+#ifdef _WIN32
+#include <windows.h>
+#endif
+
 
 PluggablePhysicsServer::PluggablePhysicsServer() {
+	#ifdef _WIN32
+	ZeroMemory(&this->fn_table, sizeof(this->fn_table));
```

Comment:
`zeromem` is defined in `core/os/copymem.h` in Godot, so I would expect it to work with MSVC too (and IIRC Godot does compile with MSVC).

Reply:
Hmm let me double check.  I had an issue with it when I tried before.

Reply:
Maybe you're running off an older version of godot?  In 3.x `zeromem` doesn't exist. `core/os/copymem.h` doesn't exist.

Reply:
https://github.com/godotengine/godot/tree/3.x/core/os I guess `zeromem` wouldn't compile on 3.x then. I can't find it in the codebase.  Should we use `memset`?

Reply:
It definitely does exist: https://github.com/godotengine/godot/blob/3.3.1-stable/core/os/copymem.h

Reply:
https://github.com/godotengine/godot/pull/48239

Reply:
They're just macros, so I'll update my patch to just use `memset`.